src/math.cpp

```diff
@@ -33,15 +29,42 @@ namespace util {
 
     double normalize_angle(double angle)
     {
-        while (angle >= M_PI)
+        double a = angle;
+
+        if (angle > pi)
```

Comment:
It's a bit of a story. When you include `<mapnik/util/math.hpp>`, which only contains one function that operates on angles, you're probably going to need `pi`, too. You don't want to include a header with a bunch of endian-conversion functions and macros. I noticed that some weeks ago and intended to move it to math.

Later I was digging in agg and saw `const double pi` there. And in C++11 we have `constexpr`. I've been using `M_PI` for years like everyone else, and was actually surprised when I learned it wasn't standardized. But `constexpr` is new, and new is always better :cocktail: